docs/docs/administer/logs.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Logs
+sidebar_position: 8
+description: Firezone's WireGuard®-based remote access platform stores debug logs and configuration, and audit logs.
+---
+
+:::note
+This guide is written for Docker based deployments of Firezone
+:::
+
+## What Firezone Logs
+
+Docker deployments of Firezone consist of 3 running containers:
+
+| Container         | Function      | Example logs                                  |
+|-------------------|---------------|-----------------------------------------------|
+| ubuntu-firezone-1 | Web portal    | HTTP requests received and responses provided |
+| ubuntu-postgres-1 | Database      |                                               |
+| ubuntu-caddy-1    | Reverse proxy |                                               |
+
+Each container stores logs as a JSON file on the host machine. These files are found at
+`var/lib/docker/containers/{CONTAINER_ID}/{CONTAINER_ID}-json.log` and can be shown through the
+`docker logs {CONTAINER}` command.
+
+## Managing and Configuring Logs
+
+By default, Firezone uses the `json-file` logging driver without
+[additional configuration](https://docs.docker.com/config/containers/logging/json-file/). This means
+logs from each container is stored separately in a file format designed to be exclusively accessed by
+the Docker daemon. Log rotation is not enabled, so logs on the host can build up and consume excess
+storage space.
+
+For production deployments of Firezone you may want to configure how logs are collected and stored. Docker
+provides [multiple mechanisms](https://docs.docker.com/config/containers/logging/configure/)
+to collect information from running containers and services.
+
+Examples of popular plugins, configurations, and use cases are:
+
+* Export container logs to
+[Splunk](https://docs.docker.com/config/containers/logging/splunk/)
+or
+[Google Cloud Logging](https://docs.docker.com/config/containers/logging/gcplogs/)
+* Enable log rotation and max file size
+* [Customize log driver output](https://docs.docker.com/config/containers/logging/log_tags/)
```

Comment:
I think this is pretty neat, and some users will find it useful. Are we advocating users send audit logs (configuration & network activity) to Splunk / GCP as well? Perhaps we should make a clearer distinction.

Reply:
Since we're not 100% sure how audit logs (config + network activity) are generated and stored at the moment, maybe we can address this later. I put the audit logs section in a separate draft doc.

If they're generated the same way as the infra/deployment logs this section would apply to all logs, so we could document everything log related on this page.

Reply:
`Infrastructure logs` or `host logs` *may* be a better term. But yeah, they're going to be completely separate from the STDOUT-based logs managed by Docker. They'll hit the database for sure -- they need to be backed up, redundant, stored with some guarantee of integrity, and possible encrypted. All things supported well by `Ecto.Schema` modules.